src/lib/draw/StreetViewMode.svelte

```diff
@@ -92,15 +80,16 @@
   }
 
   // The escape key isn't registered at all for keypress, so use keydown
+  // TODO This doesn't use eventHandler, so it might run even when the mode isn't active.
   function onKeyDown(e: KeyboardEvent) {
```

Comment:
This isn't using the new document event handler construct at all; it's bound using `svelte:window` below. Do we want to change this? If we leave it like it is now, the mode check is important. Guessing we want to migrate this to the new eventHandler though!